Python/pywarpx/picmi.py

```diff
@@ -542,6 +542,8 @@ def initialize_inputs(self):
         self.laser.beta = self.beta
         self.laser.phi2 = self.phi2
 
+        if self.fill_in:
```

Comment:
This can be simplified to:
self.laser.do_continuous_injection = self.fill_in

The picmi implementation will automatically convert True to 1.

Reply:
@dpgrote Thank you for your suggestion! Done in https://github.com/ECP-WarpX/WarpX/pull/1358/commits/62c6056ed7cc259e6180f8230c9ba0e3571756b3.